### ISSUE TITLE
About zato-web-admin service display

### ISSUE THREAD

**Issue by everraise** [zatosource/zato#1114]
Dear author, may I ask if you can screen the internal service and only display the service published by the external service?

**Comment by dsuch** [comment]
Hello,

do you mean to hide the display of internal objects, built into Zato? If so, then please send the address of the web-admin Dashboard's screen that you are thinking of.

Thank you.

**Comment by everraise** [comment]
Hello,the path is localhost:8183/zato/service/details?cluster=1,i want to hide the display of internal service.

**Comment by dsuch** [comment]
Please send in the following:

* Output from the **zato --version** command
* A screenshot showing how it looks like right now

**Comment by everraise** [comment]
it's Zato 3.1+rev.544100e-py3.6.9

**Comment by dsuch** [comment]
How did you install Zato? Using what instructions, from what URL? I ask because it looks that you have not followed all the instructions. Also, please send the screenshot.

**Comment by everraise** [comment]
Because the current working environment can only be installed offline, but using Zato is normal

**Comment by everraise** [comment]
i use rpm packages to install zato

**Comment by dsuch** [comment]
I am not sure, in your environment you do not have access to the Internet and you transported the .rpm to the server yourself and then you installed it. Is that correct?

**Comment by everraise** [comment]
also,i hope if you can pub a offline Installation Tutorials? because some people like me  the network environment can not be installed directly online

**Comment by dsuch** [comment]
Yes, this is all possible but I ask you to register on our forum here - https://forum.zato.io - where a colleague of mine will help you.

Once you are registered, please make sure to clearly describe, step by step, what you need and what you ask for. Thank you.

**Comment by everraise** [comment]
![微信图片_20210113162339](https://user-images.githubusercontent.com/54090147/104425873-25b89b00-55bc-11eb-8c1a-09e8f6104fc8.png)
How can i hide the part of the service circled in the picture.

**Comment by everraise** [comment]
I'm sorry, because I'm a Chinese user, I can't register your community account for the time being. At present, I can only communicate with you on GitHub, please forgive me.

**Comment by dsuch** [comment]
It is fine, do not worry. As far as hiding these internal services, it is not currently possible but I will think of something and let you know.

As far as building your own documentation goes - you can find it here https://github.com/zatosource/zato-docs.

But you need to understand that you did not install Zato fully. You only have a base package and there have been hundreds of new updates since the .rpm was produced.

This is why I ask again for confirmation how you installed Zato. Did you upload the .rpm to your server? Does your server have a connection to the Internet?

**Comment by everraise** [comment]
hello,at present, the working environment is not allowed to connect to the Internet, so it can only be used in the LAN, so I downloaded Zato locally, compressed it into RPM package with yum --downloadly command, and then deployed it to the CentOS server. Related dependencies also exist and can run normally.

**Comment by everraise** [comment]
You are a very enthusiastic author. I am very happy to communicate with you. I hope I can work with you in the future.

**Comment by dsuch** [comment]
In that case, you only have a base package - since its release dozens and hundreds of new commits were added so for sure what you have will not suffice.

I will ask a colleague to contact you and explain how to build your own package with all the commits for you to install it in your server.

Yes, I am also happy to help you.

**Comment by everraise** [comment]
thank you !

**Comment by DimShadoWWW** [comment]
Hello @everraise,

To build your own .rpm from Zato's source code you can follow this steps:

1. - clone https://github.com/zatosource/zato-build
2. enter the folder of your system: rhel7 (Centos7) or rhel8 (Centos8)
3. run `./build-zato.sh support/3.1 3.1.0 python3`

At the end of the process, you should see the .rpm file. I hope these instructions are helpful to you.

regards,

**Comment by everraise** [comment]
Thank you very much! I'll try and let you know if there are any questions.

**Comment by everraise** [comment]
dear author，i want to know how to do cross-hardware load balance cluster.

**Comment by dsuch** [comment]
Hello @everraise - this is not how Open Source works. You cannot open one ticket, ask questions, do not provide feedback and then ask more, unrelated questions in the same ticket. This only leads to confusion and disorder.

Instead, here is what you need to do:

* Tell @DimShadoWWW whether the instructions provided helped you or not
* Open a new ticket if you want to ask a new question
* In this new ticket, you need to provide detailed information what you want to achieve. Also, you need to say what you already tried in order to achieve it.

In short, everything needs to be clean, tidy and fully explained. You cannot just ask one sentence questions and expect that people will be eager to help you, this is simply not how it works.

**Comment by everraise** [comment]
Hello, my problem has been solved. Thank you for your help
